alerts/couchbase/spiking-evictions.json

```diff
@@ -0,0 +1,36 @@
+{
+    "displayName": "Couchbase - Spiking Ejections",
+    "documentation": {
+        "content": "If `bucket.item.ejection.count` begins to spike this could indicate that there is unexpected memory pressure. The default threshold is set to 10 for the alert policy and this can be customized for your environment.",
+        "mimeType": "text/markdown"
+    },
+    "userLabels": {},
+    "conditions": [
+        {
+            "displayName": "VM Instance - workload/couchbase.bucket.item.ejection.count",
+            "conditionThreshold": {
+                "filter": "resource.type = \"gce_instance\" AND metric.type = \"workload.googleapis.com/couchbase.bucket.item.ejection.count\"",
+                "aggregations": [
+                    {
+                        "alignmentPeriod": "300s",
+                        "crossSeriesReducer": "REDUCE_NONE",
+                        "perSeriesAligner": "ALIGN_MEAN"
+                    }
+                ],
+                "comparison": "COMPARISON_GT",
+                "duration": "30s",
```

Comment:
condition_threshold.duration had an invalid value of \"30s\": only multiples of 1 minute are supported